src/client/kinto_http.rs

```diff
@@ -48,18 +58,19 @@ impl From<std::num::ParseIntError> for KintoError {
     }
 }
 
-pub async fn get_records(
+pub async fn get_last_modified_change(
```

Comment:
I liked your naming for `LatestChangeResponse`. Maybe this could be `get_latest_change_timestamp` ?

Reply:
done